Add detailed installation instructions for frontend and backend

medelm1 opened the pull request:
This pull request adds comprehensive installation instructions to the README.md file. The following sections have been added:

1. Steps on forking the project.
2. Installation instructions for the frontend.
3. Installation instructions for the backend.
4. Instructions on how to run the application.

These additions will help new contributors set up the project more easily.

**Changes:**
- Updated README.md with detailed steps for setting up the development environment.
- Added prerequisites and environment variable setup instructions.

Please review and let me know if any further adjustments are needed. Thank you!

alvarotorrestx commented on README.md:
````diff
@@ -32,7 +32,73 @@ DevConnect is an open-source platform designed to connect developers from around
+3. Clone your forked repository to your local machine:
+
+```bash
+git clone https://github.com/alvarotorrestx/DevConnect.git
````
Please update the "alvarotorrestx" to "yourgithubprofile" so it's more general and people don't get confused

alvarotorrestx commented on README.md:
````diff
@@ -32,7 +32,73 @@ DevConnect is an open-source platform designed to connect developers from around
+cd frontend
+npm run dev
+```
+
````
Can you add or update to differentiate frontend to be port 5173

alvarotorrestx reviewed (commented):
Overall great job, I love the layout!

I just added some minor changes to be made, please.

Thank you for contributing! 🎇

medelm1 commented:
Sure! Rest assured.

medelm1 commented:
The following changes have been made:

1. Updated the forking instructions to replace "alvarotorrestx" with "yourgithubprofile" to avoid confusion.
2. Added a note to specify that the frontend runs on port 5173.
3. Added a note to specify that the backend runs on port 3000.

Please review and let me know if any further adjustments are needed. Thank you!
